jdk/make/CopyFiles.gmk

```diff
@@ -321,28 +338,11 @@ else
   $(JVMCFG):
 	$(MKDIR) -p $(@D)
 	$(RM) $(@)
-        # Now check for other permutations
-        ifeq ($(JVM_VARIANT_SERVER), true)
-	  $(PRINTF) "-server KNOWN\n">>$(@)
-	  $(PRINTF) "-client ALIASED_TO -server\n">>$(@)
-          ifeq ($(JVM_VARIANT_MINIMAL1), true)
-	    $(PRINTF) "-minimal KNOWN\n">>$(@)
-          endif
-        else
-          ifeq ($(JVM_VARIANT_CLIENT), true)
-	    $(PRINTF) "-client KNOWN\n">>$(@)
-	    $(PRINTF) "-server ALIASED_TO -client\n">>$(@)
-            ifeq ($(JVM_VARIANT_MINIMAL1), true)
-	      $(PRINTF) "-minimal KNOWN\n">>$(@)
-            endif
-          else
-            ifeq ($(JVM_VARIANT_MINIMAL1), true)
-	      $(PRINTF) "-minimal KNOWN\n">>$(@)
-	      $(PRINTF) "-server ALIASED_TO -minimal\n">>$(@)
-	      $(PRINTF) "-client ALIASED_TO -minimal\n">>$(@)
-            endif
-          endif
-        endif
+	# Now j9vm is the only place to locate jvm.dll for any mode
+	$(PRINTF) "-j9vm KNOWN\n">>$(@)
+	$(PRINTF) "-client IGNORE\n">>$(@)
```

Comment:
Might as well remove everything but j9vm since none of the others will work with OpenJ9

Reply:
Removed all "IGNORE" items.